citi-onboarding/pride-hack
# feature(apiNotice): url edital

## What I did
* Server updates
* Added a /views/Notice and /model/Notice.js
* Included Notice requires to routes/indes.js

## How to test
1. To test access localhost:3001/admin/ and access the KeystoneJS
2. Go to section Notice and add a new URL
3. If the database works, you must see it at localhost:3001/api/notice
